atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/ColumnRangeSelections.java

```diff
@@ -20,9 +20,13 @@
 import com.google.common.base.Preconditions;
 import com.palantir.common.persist.Persistable;
 
-public class ColumnRangeSelections implements Serializable {
+public final class ColumnRangeSelections implements Serializable {
+    private ColumnRangeSelections() {
+        // Utility class
+    }
+
     public static ColumnRangeSelection createPrefixRange(byte[] prefix, int batchSize) {
-        byte[] startCol = Preconditions.checkNotNull(prefix).clone();
+        byte[] startCol = Preconditions.checkNotNull(prefix, "prefix cannot be null").clone();
```

Comment:
Yes, I agree. These error messages should be much better than what I inserted but there were so many it would've taken way too long to write nice messages for each one now. When we encounter them, we should try to make them nicer though.